src/time.js

```diff
@@ -9,6 +9,20 @@ function advanceBlock () {
   });
 }
 
+// Advance the block to the passed height
+async function advanceBlockTo (target) {
+  if (!BN.isBN(target)) {
+    target = new BN(target);
+  }
+
+  let now = (await latestBlock());
+  if (target.lt(now)) throw Error(`Cannot advance current block (${now}) to a moment in the past (${target})`);
```

Comment:
The error message is misleading: `target` is not a moment in the past but a block number lower than the current one